feature/container/containerz/tests/container_lifecycle/containerz_test.go

```diff
@@ -0,0 +1,66 @@
+package container_lifecycle
+
+import (
+	"context"
+	"crypto/tls"
+	"flag"
+	"testing"
+
+	"github.com/openconfig/containerz/client"
+	cpb "github.com/openconfig/featureprofiles/internal/cntrsrv/proto/cntr"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
+)
+
+var (
+	containerTar   = flag.String("container_tar", "/tmp/cntrsrv.tar", "The container tarball to deploy.")
+	containerzAddr = flag.String("containerz_addr", "localhost:19999", "containerz server address")
```

Comment:
Should the default address here not be the address of the DUT per the comment below too?